modules/nw-ovn-kubernetes-logs.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-logs_{context}"]
+= OVN-Kubernetes logs
+
+You can view the logs for each of the pods in the `ovnkube-master` and `ovnkube-node` pods in the OpenShift CLI (`oc`) and web console.
+
+.Prerequisites
+
+* Access to the OpenShift CLI (`oc`).
+
+.Procedure (UI)
```

Comment:
These are really two different modules--one for performing this procedure with a web UI, and the other with a CLI.

Reply:
Done